src/dotnet-install.sh

```diff
@@ -1359,7 +1394,9 @@ do
         --feed-credential|-[Ff]eed[Cc]redential)
             shift
             feed_credential="$1"
-            non_dynamic_parameters+=" $name "\""$1"\"""
+            #feed_credential should start with "?", for it to be added to the end of the link.
+            #adding "?" at the beginning of the feed_credential if needed.
+            [[ -z "$(echo $feed_credential)" ]] || [[ $feed_credential == \?* ]] || feed_credential="?$feed_credential"
```

Comment:
Curious why the `echo` is needed. Couldn't we just do:
```suggestion
            [[ -z "$feed_credential" ]] || [[ $feed_credential == \?* ]] || feed_credential="?$feed_credential"
```

Reply:
This is to have the same behavior as .ps1 script. In powershell we have 
`[string]::IsNullOrWhitespace($FeedCredential)`.  
To do the same in bash we trim the value with echo and then check if it is an empty string.

Reply:
Neat! I didn't know about this new trick, thank you.